README.md

```diff
@@ -1 +1,107 @@
+
+The goal of a team is to produce meaningful, high quality work at a predictable [pace](#notes)<sup>2</sup>.
+ 
+##Workflow
+ 
+Workflow is a structured approach to how a team evolves an idea from inception to production. Teams evolve each idea one function at a time, with each function’s output being the next function’s input. 
+
+![Workflow and deliverables](https://github.com/vimeo/Blueprint/blob/working-draft/assets/workflow.png)    
+<sup>*Workflow and deliverables*</sup>
+
+Consider an example where the idea we're working on is captured by this user story: *"As a user, I'd like to be able to upload videos to Vimeo from within the Vimeo iOS app."* The team's product manager creates a feature spec. The team's designer uses this spec to generate visuals. The team's developer uses the product and design documentation to build the upload feature. And the team's QA representative tests the resulting build and documents any bugs that surface. Based on QA's test results we determine whether the build is shippable. 
+
+Each function's output must be reviewed and approved by the other functions before it can advance to the next stage. This helps us make decisions when we have enough info to make them (not before, not after). It also ensures cross-functional ownership at each stage.
+
+This is our ideal workflow. In practice the boundaries are softer and the communication more fluid. No matter how the workflow is applied the emphasis is on discussion across functions and platforms early and often (e.g. product specs out a feature, design identifies potential conceptual inconsistencies, engineering identifies technical limitations, product updates the spec, and so on). 
+
+These checks and balances increase our chances of success and predictability by helping us abstract and define problems better and therefore making us less likely to encounter issues down the line. However, problems are not always avoidable and this is where a willingness to engage in iterative design is important.
+
+We solicit feedback early, ask questions often, and [document and share what we learn](#notes)<sup>3</sup> along the way. 
+
+##Sprints
+ 
+Sprints are structured 2-week intervals at which a team executes the above workflow to evolve ideas. Sprints help teams establish [pace](#notes)<sup>2</sup>. Pace is valuable because it allows us to predict how long things will take in the short and long term. 
+ 
+![Sprint workflow](https://github.com/vimeo/Blueprint/blob/working-draft/assets/workflow-with-sprints.png)    
+<sup>*Sprint workflow*</sup>
+
+Before beginning a sprint:
+ 
+1. We assemble a rough list of stories to be included in the sprint (a mix of product, design, dev, and qa stories),
+2. We determine who will work on each story (via self-selection rather than top-down assignment), 
+3. Assignees break each of their stories into small, actionable chunks,
```

Comment:
We could say "as tickets" unless thats too specific to Jira. Maybe add like "(as a ticket, for example)" to emphasize that this is a tangible chunk not just a mental chunk in the developers head